pkg/agent/bakerapi.go

```diff
@@ -130,3 +140,23 @@ func (agentBaker *agentBakerImpl) GetDistroSigImageConfig(
 
 	return allDistros, nil
 }
+
+func (agentBaker *agentBakerImpl) GetCachedVHDImages() ([]string, error) {
+	pwd, err := os.Getwd()
```

Comment:
we can probably just hard-code a file path here instead of always resolving cwd, since that shouldn't ever change at runtime